repose-aggregator/src/docs/asciidoc/recipes/getting-started.adoc

```diff
@@ -62,56 +62,60 @@ Following are some of the most common ones:
 Then, configure your origin service endpoint in the <<../architecture/system-model.adoc#,system model>>, and you are ready to configure <<../filters/index.adoc#,filters>> and <<../services/index.adoc#,services>> that work for you.
 We have many sample configurations that you can use to get started.
-Or, because Repose is open source, you can <<custom-filter-for-repose.adoc#,build your own>> stacks of reusable software components.
+Or, because *Repose* is open source, you can <<custom-filter-for-repose.adoc#,build your own>> stacks of reusable software components.
 
 === Environment Setup
 Installation instructions are provided for Debian package managers (e.g., APT) and RPM package managers (e.g., Yum).
 
 Certain deployment methods will handle environment setup automatically.
 
 === Installation
-The most common and preferred way to run Repose is as a standalone application using the Valve installation.
-This installation allows Repose to be executed either as a Linux service (e.g. systemd) or directly from the installed JAR.
-An alternative would be to run Repose inside an existing servlet container (e.g. Tomcat, GlassFish) using the WAR installation.
+The recommended and most common way to run *Repose* is as a standalone application using the Valve installation.
+This installation allows *Repose* to be executed either as a Linux service (e.g. systemd) or directly from the installed JAR.
+An alternative would be to run *Repose* inside an existing servlet container (e.g. Tomcat, GlassFish) using the WAR installation.
 
-While Repose can be installed manually, there are other deployment options available.
+The most recent addition to the *Repose* family is a containerized release of the Valve installation.
+This is great if you are looking to eventually run *Repose* with a container service or are just trying it out for the first time.
+
+While *Repose* can be installed manually, there are other deployment options available.
 However, these other methods do require a little more System Administration knowledge which is not covered here.
 See: <<Deployment>>
 
 ==== Proxy Server (Valve)
 This is the most common setup and is detailed on the <<valve-installation.adoc#,Valve Installation>> page.
 
-After installing it this way, consult the <<running-valve.adoc#,Running Valve>> page for details on starting and stopping Repose.
+After installing it this way, consult the <<running-valve.adoc#,Running Valve>> page for details on starting and stopping *Repose*.
 
 ==== Servlet Container WAR
 Instructions for installing the WAR are on the <<war-installation.adoc#,WAR Installation>> page.
 
 ===== Tomcat Setup
-
 After installing the WAR, there is additional setup to get everything working with Tomcat.
 See <<tomcat-setup.adoc#,Tomcat Setup>> for more details.
 
 ===== GlassFish Setup
-
 See <<glassfish-setup.adoc#,GlassFish Setup>> for more details.
 
 ===== Container Attributes known to affect Repose
-
```

Comment:
Should this be in the style guide? No blank lines after section headers?

Reply:
Adding it now.

Reply:
Added.